README.md

```diff
@@ -100,7 +100,7 @@ Pull requests are welcome, please see [the contribution guidelines](CONTRIBUTING
 # Getting Started
 
 - [bitcoincash.org](https://bitcoincash.org) - A general multi-lingual introduction.
-- [BCH Info](https://bch.info/) - Multilingual site for general information about bitcoin cash.
+- [BCH Info](https://bch.info/en/) - Multilingual site for general information about bitcoin cash.
```

Comment:
They are not doing that - at least https://bch.info/ and https://bitcoincashnode.org/

My German language setting does not seem to change the language there.

Country-based detection is bad, because a VPN / Proxy / Tor changes the GeoIP and some countries have several languages depending on the area within the country.